site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -118,11 +118,31 @@ Next, make sure that you can authenticate using gcloud:
 ```
 gcloud auth application-default login
 ```
-
+{{% feature expiryVersion="1.9.0" %}}
 Now you can create your GKE cluster (optionally specifying the version of Agones you want to use):
 ```
 terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.0.0"]
 ```
+{{% /feature %}}
+
+{{% feature publishVersion="1.9.0" %}}
+#### Option 1: Creating the cluster in the default VPC
+To create your GKE cluster in the default VPC just specify the project variable.
+
+(optionally you can specify the version of Agones you want to use):
+```
+terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.7.0"]
```

Comment:
@markmandel - should we add this variable to the release checklist so that it stays up to date?

Reply:
Hi @roberthbailey , updated!